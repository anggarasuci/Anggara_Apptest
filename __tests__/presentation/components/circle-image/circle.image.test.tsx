import React from 'react';
import {render} from '@testing-library/react-native';
import CircleImageView from '../../../../src/presentation/components/circle-image/circle.image';

test('renders CircleImageView correctly', () => {
  const {toJSON} = render(
    <CircleImageView url="https://example.com/image.jpg" />,
  );

  expect(toJSON()).toMatchSnapshot();
});
