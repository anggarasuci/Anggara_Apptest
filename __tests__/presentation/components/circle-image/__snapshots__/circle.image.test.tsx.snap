// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CircleImageView correctly 1`] = `
<View
  style={
    {
      "borderRadius": 50,
      "height": 100,
      "overflow": "hidden",
      "width": 100,
    }
  }
>
  <Image
    source={
      {
        "uri": "https://example.com/image.jpg",
      }
    }
    style={
      {
        "flex": 1,
        "height": null,
        "resizeMode": "cover",
        "width": null,
      }
    }
  />
</View>
`;
